src/main/java/seedu/duke/command/FindCommand.java

```diff
@@ -36,13 +37,15 @@ public FindCommand(String keywords) {
         this.keywords = keywords;
     }
 
+
     /**
      * Checks the format of find to ensure that it contains keywords used in the search expression.
      *
      * @param keywords A string containing the keywords used in the search expression.
      * @throws FindTransactionMissingKeywordsException If a user does not enter a search expression for Find.
      */
     public static void checkFindFormat(String keywords) throws FindTransactionMissingKeywordsException {
+        findLogger.setLevel(Level.SEVERE);

```

Comment:
Thanks for adding this set level statement here to control the display of the severity message!